pages/about.vue

```diff
@@ -95,6 +95,20 @@ export default {
   background-image: url('@/static/image/chapter_intro_background.jpg');
   background-size: cover;
   color: $white-color;
+  padding: $offset-3;
+
+  @media (max-width: $query-mobile) {
+    background-image: url('@/static/image/about_page_background_mobile.jpg');
```

Comment:
the approach of a separate (extra tall) bg image for a page that runs longer vertically seems strange, especially since it has the same content as `static/image/chapter_intro_background_mobile.jpg`.
I would have expected a solution that reuses the existing intro bg image, e.g. by tiling it, or by `cover`ing the screen and keeping its position fixed (or perhaps another solution).

Besides the additional loading of the image straining bandwidth and thus performance, it doesn't cause any real issues, so no need to fix at this point, but I wanted to put my thoughts into the review, since we're both learning here :)

Reply:
My reasoning is that having a separate image cropped for mobile actually saves bandwidth since the cropped images are all smaller in size than the original images. However this is only a theory as I'm not sure if images that fall outside of the media query aren't loaded as well.

In addition i had trouble making the portal in the original artwork centered on mobile since it's not centered in the original artwork